README.md

```diff
@@ -161,7 +161,7 @@ Apps people use when taking transit.
 - [Graphs in Transit](https://gtfs-graph.herokuapp.com/) - Graph centrality metrics displayed over rapid transit networks for New York City, Boston, and Paris.
 - [SEPTA Rail OTP Report](https://github.com/fulldecent/septa-regionalrail-otp) - An online on-time performance reporing & drill down tool using GTFS.
 - [TransitFlow](https://github.com/transitland/transitland-processing-animation) Animate GTFS data around the world using Processing and Transitland.
-- [gtfspy-webviz](https://github.com/CxAalto/gtfspy-webviz) - Web application for animation and visualization of GTFS data using the [gtfspy](https://github.com/CxAalto/gtfspy) Python3 library.
```

Comment:
So from a quick look I think the `gtfspy-webviz` should stay here, and `gtfspy` should be added to the "GTFS Analysis Tools" (https://github.com/CUTR-at-USF/awesome-transit#gtfs-analysis-tools) section (and it should have a link to gtfs-py-webviz, just like `gtfspy-webviz` has a link to `gtfspy`).

Does that make sense to you?

Reply:
Yeah sounds good to me.